qtt/measurements/scans.py

```diff
@@ -494,6 +494,21 @@ def scan1Dfast(station, scanjob, location=None, liveplotwindow=None, delete=True
     t0 = time.time()
     wait_time_startscan = scanjob.get('wait_time_startscan', 0)
 
+    try:
```

Comment:
You could turn this addition into a (private) function and use it in the `scan1Dfast` and `scan2DFast` function (dry principle).